### PR TITLE
Wip

### PR DESCRIPTION
**Pull request by rhydvik**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -38,11 +38,10 @@
...
-     }
-}
-
+  "jest": {
+    "moduleNameMapper": {
+      "\\.(css|less|sass|scss)$": "<rootDir>/__mocks__/styleMock.js",
+      "\\.(gif|ttf|eot|svg)$": "<rootDir>/__mocks__/fileMock.js"
+    }
+  }
 }
```
Comment (vipul-21) [inline package.json:47]: @rhydvik Nothing new here, right (just which space)?

File: .gitignore [hunk 2]
```diff
@@ -1,2 +1,3 @@
 /node_modules
 npm-debug.log*
+.idea/*
```
Comment (vipul-21) [inline .gitignore:3]: `.idea` should be ignored. Remove `+`.

Comment (vipul-21) [reply .gitignore:3]: Although out of curiosity, what is `idea` ?

Comment (rhydvik) [reply .gitignore:3]: Idea contains some web-storm configurations.

**Pull request closed without merging**